spec/integration/task_spec.cr

```diff
@@ -0,0 +1,24 @@
+require "../spec_helper"
+
+describe "Running a task" do
+  it "returns a non-zero exit code when running a missing task" do
+    task("missing.task").exit_status.should be_not_found
+  end
+end
+
+private def run(process)
+  Process.run(
+    process,
+    shell: true,
+    output: true,
+    error: true
+  )
+end
+
+private def task(task_name)
+  run("crystal spec/tasks.cr --no-debug -- #{task_name}")
+end
+
+private def be_not_found
+  eq 127
```

Comment:
Since we're not 100% sure about the status code, and I think as long as we test that the exit wasn't `0` what if we change this to `should_not be_successful` and this helper would be `eq 0`

I think that will fix the test, and it will be less coupled to the exact status code, which is good since the most important part is that it is not 0

Reply:
Sure thing!